tfrecord/reader.py

```diff
@@ -326,7 +365,7 @@ def tfrecord_loader(data_path: str,
     """
     if sequence_description is not None:
         return sequence_loader(data_path, index_path, description, sequence_description, shard)
```

Comment:
Pass compression_type to sequence_loader as well.

Reply:
Done 👍